src/main/java/com/zendesk/maxwell/producer/partitioners/AbstractMaxwellKafkaPartitioner.java

```diff
@@ -0,0 +1,12 @@
+package com.zendesk.maxwell.producer.partitioners;
+
+import com.zendesk.maxwell.row.RowMap;
+
+public abstract class AbstractMaxwellKafkaPartitioner extends AbstractMaxwellPartitioner {
+
+	public AbstractMaxwellKafkaPartitioner(String partitionKey, String csvPartitionColumns, String partitionKeyFallback) {
+		super(partitionKey, csvPartitionColumns, partitionKeyFallback);
+	}
+
+	public abstract int kafkaPartition(RowMap r, int numPartitions);
```

Comment:
this looks like maybe `AbstractMaxwellKafkaPartitioner` wants to be an interface rather than an abstract class

Reply:
or nothing whatsoever, it'd be fine to have the two kafka partitioners inherit directly from  `AbstractMaxwellPartitioner`